compair/models/assignment.py

```diff
@@ -68,6 +71,7 @@ class Assignment(DefaultTableMixin, UUIDMixin, ActiveMixin, WriteTrackingMixin):
 
     # hybrid and other functions
     course_uuid = association_proxy('course', 'uuid')
+    course_end_date = association_proxy('course', 'end_date')
```

Comment:
can be removed since we only need this in the time period validator

Reply:
More defensive programming - removing.